docs/reference/ilm/overview-ilm.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[testenv="basic"]
+[[overview-index-lifecycle-management]]
+== Index lifecycle management overview
+
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+You can create and apply {ilm-cap} ({ilm-init}) policies to automatically manage your indices 
+according to your performance, resiliency, and retention requirements. 
+Index lifecycle policies can trigger actions such as rollover, shrink, freeze, and delete. 
+
+Typically, you associate a lifecycle policy with an index template so it is applied   
+whenever an index is created from the template. 
+You can also apply a policy manually when you create an index.
+
+Index lifecycle policies are most often used to manage time-series indices. 
+Time-series indices have a four stage lifecycle:
+
+* Hot--the index is actively being updated and queried.
+* Warm--the index is no longer being updated, but is still being queried.
+* Cold--the index is no longer being updated and is seldom queried. The
+information still needs to be searchable, but it's okay if those queries are
+slower.
+* Delete--the index is no longer needed and can safely be deleted.
+
+A lifecycle policy governs how the index transitions through these stages and
```

Comment:
```suggestion
You can use a lifecycle policy to govern how an index transitions through these stages and
```

Reply:
I tweaked this a bit, but left the focus on the policy driving the transitions and actions.